sphinx/source/docs/user_guide/server.rst

```diff
@@ -1430,55 +1408,57 @@ Then in your web application, we explicitly provide (signed) session ids using
                             session_id=generate_session_id())
     return render_template("embed.html", script=script, template="Flask")
 
-Make sure that the ``BOKEH_SECRET_KEY`` environment variable is set (and
-identical) for both the Bokeh server and web app processes (e.g. Flask or
-Django or whatever tool is in use).
+Make sure to set identical ``BOKEH_SECRET_KEY`` environment variables both for
+the Bokeh server and for the web app processes, be it Flask, Django, or any
+other tool you are using.
 
 .. note::
 
-    Signed session IDs are effectively access tokens. As with any token system,
-    security is predicated on keeping the token a secret. It is also advised to
-    run the Bokeh server behind a proxy that terminates SSL, so that the session
-    ID is transmitted securely to the user's browser.
+    Signed session IDs serve as access tokens. As with any token system,
+    security is predicated on keeping the token secret. You should also run
+    the Bokeh server behind a proxy that terminates SSL connections. This lets
```

Comment:
Bokeh server can now terminate its own SSL connections so maybe add ", or configure the Bokeh server to terminate SSL directly"